### PR TITLE
style: cleanup files

### PR DESCRIPTION
**Pull request by kenji-miyake**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # pre-commit-hooks-ros
 
-ros-related hooks for pre-commit <https://pre-commit.com/>
+[ROS][]-related hooks for [pre-commit]
```
Comment (kenji-miyake) [inline README.md:3]: Ah, mistake. :pleading_face: 

```suggestion
[ROS][]-related hooks for [pre-commit][]
```

Comment (kenji-miyake) [reply README.md:3]: But without `[]` works as well? :thinking:

Comment (kenji-miyake) [reply README.md:3]: Will check the official reference.

Comment (kenji-miyake) [reply README.md:3]: I'll remove `[]`.

https://spec.commonmark.org/0.30/#example-556
https://github.github.com/gfm/#example-565

**Review by KeisukeShima — APPROVED** [review]
LGTM

**Pull request merged**